content/docs/next-release/frontend-ui.md

```diff
@@ -12,7 +12,8 @@ Several aspects of the UI can be configured:
   * The Dependencies section can be enabled / configured
   * Google Analytics tracking can be enabled / configured
   * Additional menu options can be added to the global nav
-
+  * Search input can be configured
```

Comment:
```suggestion
  * Search defaults can be configured
```

Reply:
In this case, it isn't a default value, it's the max value that the input(maxLimit and maxLookback) can take.

Reply:
then let's change to "search input limits", because we're not configuring "inputs"